Sources/MediaCategoryCells/ArtistCollectionViewCell.swift

```diff
@@ -49,13 +46,25 @@ class ArtistCollectionViewCell: BaseCollectionViewCell {
     }
 
     override class func cellSizeForWidth(_ width: CGFloat) -> CGSize {
-        let numberOfCells: CGFloat = round(width / 320)
+        let numberOfCells: CGFloat
+        if width <= 414 { //max iPhone portrait width
+            numberOfCells = 1
+        } else if width <= 896 { // max landscape iPhone width includes small iPads in protrait
+            numberOfCells = 2
+        } else if width <= 1024 { // max landscape for small iPads
+            numberOfCells = 3
+        } else { //ipad Pro in Landscape
+            numberOfCells = 4
+        }
+
```

Comment:
I would put this logic into a helper method inside `BaseCollectionViewCell` since it looks like the values constant.

Also, we could consider having an enum for the values(414, 896, 1024). It could be a nice improvement for readability and usability(for example we could switch over it).

Reply:
I was actually thinking about merging the artist and audio cell, then you avoid a lot more of duplication. numberOfCells differ between audio and video so I think BasecollectionViewCell might not work but please correct me here if I'm wrong. 
I like the enum idea a lot since these numbers are shared but I would want to look into .compact and alike. Is it okay if I make merging cells and enum a follow up and separate PR?

Reply:
Ideally, I would say that at least the enum should be part of this PR, but in the end, I don't mind either way. 

Your call~